lib/services/catalog/index.js

```diff
@@ -136,6 +138,16 @@ module.exports = function (bus, options) {
 				}}),
 				CatalogItemController.create({bus, type})
 			);
+
+			if (type === 'collection') {
```

Comment:
At this point I've only added this for collections.  Wasn't sure what other types would get this route available to them.